script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -226,48 +235,51 @@ But the most abnormal thing of all is―――
 [sha:1cb63ee6c72033905af9acb28650baff38a63130]{
 -- Page 273, Offset 32421.
 -- その“異物”から放たれる、殺意とも敵意ともとれない、ドロドロに入り乱れた“意識”だった。
-It is the consciousness emitting out from its body; a muddy mixture that could not be perceived as purely murderous or hostile.
+Is the murky and jumbled 'consciousness' being given off by that aberration,
+ which she can't pin down as either hostility or outright bloodlust.
 }
 [sha:43e5119cd762725f4404aec63d9205db64686f63]{
 -- Page 274, Offset 32422.
 -- 向けられるだけで死を覚悟する殺意は知っている。
-Just by being looked at, she could feel its murderous intentto the extent that she was prepared to die.
+Faced by it, she senses a murderous impulse strong enough to make one curl up and accept death.
 }
 [sha:039195b7f185e5f72ce484bb37ee6046dbfc5842]{
 -- Page 274, Offset 32423.
 -- 向けられるだけで全身が粟立つ敵意は知っている。
-Just by being looked at, she could feel its hostility to the extent that she feels goosebumps all over her body.
+Faced by it, she senses hostility so extreme it makes her break out in goosebumps all over her body.
 }
 [sha:656bc88dbb4aaa14f664446f72e984bf5399f95a]{
 -- Page 274, Offset 32424.
 -- だがこの意識は経験にない。解析する術もない。
-But this awareness has never been experienced before. There's no way of analyzing it either.
+But she has never run across a consciousness like this before.
+ She has no way to analyze it.
 }
 [sha:0038bd8d325448a292d8da22a0d854d59c0df0e4]{
 -- Page 274, Offset 32425.
 -- きっと、教会のどの資料にも記されてすらいない。
-Neither is there any reason to believe that this is written in any of the Church's documents.
+This thing probably isn't even written of in any of the Church's documents.
 }
 [sha:c31ca372324d5624b057ad32442044f17e746a16]{
 -- Page 275, Offset 32426.
 -- この“意識”が何に起因するものなのか、常人には不理解だ。
-// i have no idea what this narrator asshole has been spewing
-It is incomprehensible for ordinary humans to find the cause of this new-found 'consciousness'.
+No ordinary human could comprehend a creature capable of giving rise to such a 'consciousness'.
 }
 [sha:d2f9141c9253570357d23293aabb084bd7465909]{
 -- Page 275, Offset 32427.
 -- とある高名な画家の油絵を思い出す。
+// とある画家のファン・ゴッホ
```

Comment:
this one right
<img width="930" alt="image" src="https://user-images.githubusercontent.com/16092757/169901262-003b47cc-99bc-401b-b3f6-8fa107b8bb7a.png">